src/it/scala/io/iohk/ethereum/sync/FastSyncItSpec.scala

```diff
@@ -180,6 +182,65 @@ class FastSyncItSpec extends FlatSpecBase with Matchers with BeforeAndAfterAll {
     }
   }
 
+  it should "switch to regular sync once `safeDownloadTarget` is reached" in customTestCaseResourceM(
```

Comment:
Where do you assert that regular sync was initiated?

Reply:
the `CommonFakePeer#waitForFastSyncFinish()` helper method checks for the completion of Fastsync. More specific assertion can be made possible by intercepting the logs only. Maybe something to improve in another ticket to add support for intercepting logs in tests.